File: loianeTraining/javaBasico/exercicios-aulas-14-15/Ex2.java
import java.util.Scanner;

class Ex2 {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		System.out.println("Digite um numero");
		int number = scan.nextInt();

		if (number > 0) {
			System.out.println("Positivo!");
		} else if (number == 0) {
			System.out.println("Neutro!");
		} else {
			System.out.println("Negativo!");
		}
		scan.close();
	}
}

File: programando-seu-futuro/Extra/2024/Modulo1/Lista5/Ex2.cs
using System;

class Ex2
{
    static void Main(string[] args)
    {
        for (int i = 0; i <= 10; i++)
        {
            for (int j = 0; j <= 10; j++)
            {

                Console.WriteLine("{0} X {1} = {2}", i, j, i * j);
            }
            Console.WriteLine();
        }
    }
}